<?xml version="1.0"?>
<launch>
  <node name="netft_node" pkg="netft_utils" type="netft_node" args="10.10.0.245"/>
  <node name="netft_utils" pkg="netft_utils" type="netft_utils" args="world world"/>
</launch>